src/features/game/expansion/components/resources/Stone.tsx

```diff
@@ -276,6 +245,23 @@ export const Stone: React.FC<Props> = ({ rockIndex, expansionIndex }) => {
               width: `${GRID_WIDTH_PX}px`,
             }}
           />
+          <ResourceDropAnimator
+            wrapperClassName="h-full w-full"
+            mainImageProps={{
+              src: hitbox,
+              className: "absolute opacity-50",
+              style: {
+                width: `${GRID_WIDTH_PX}px`,
+              },
+            }}
+            dropImageProps={{
+              src: stone,
+              className: "w-8",
+            }}
```

Comment:
To ensure pixel sizes are the same, we can use the actual width of the Stone image (10px wide)

E.g.
```
            dropImageProps={{
              src: stone,
              style: {
                width: `${PIXEL_SCALE * 10}px`,
              },
            }}
```